kinds/dashboard/dashboard_kind.cue

```diff
@@ -74,7 +74,7 @@ lineage: schemas: [{
 
 			// Version of the JSON schema, incremented each time a Grafana update brings
 			// changes to said schema.
-			schemaVersion: uint16 | *41
+			schemaVersion: 41 // The plan is to send everyone else down to v0 or have them upgrade to v2.
```

Comment:
I'm not quite sure we can use it here without breaking existing code, because IIRC this is used for existing code in the UI.

If we want to validate minimum schema version, we might want to do that as a separate v1 check on top of regular CUE schema validation.

Also, I _think_ we can run a bunch of schema migrations in the backend for V1 – @stephaniehingtgen has a PR somewhere, so this number should be something like `36`.

@grafana/dashboards-squad correct me if I'm wrong.

Reply:
👋 Yes @radiohead, we can already run some backend migrations. We perform this check to determine if we have the necessary migrations in place; otherwise, we return the dashboard as it is: https://github.com/grafana/grafana/pull/102357. 

The schemaVersion can be any number between 0 and 41. 

When it comes from the frontend, it must be 41. 

However, when requesting a dashboard from the backend, it can be any `schemaVersion` under 37. If it is greater than 36, it will run the migrations in the backend and return 41.

Reply:
Got it, i will change this back and add an explicit check for schemaVersion == 41 for v1 in the validation. Ok?

My understanding was that we don't want anyone but the dashboardservice and /api to send us old dashboards (done via the FieldValidation: Ignore) to the v1 endpoint. Is that understanding wrong?

Reply:
Changed and moved this into the validator